packages/contracts-bedrock/test/dispute/FaultDisputeGameN.t.sol

```diff
@@ -462,22 +462,22 @@ contract FaultDisputeGameN_Test is FaultDisputeGame_Init {
         claim = _changeClaimStatus(claim, VMStatuses.PANIC);
 
         vm.warp(block.timestamp + 10);
-        bond = _getRequiredBond(2);
+        bond = _getRequiredBondV2(2, 0);
         (,,,, disputed,,) = gameProxy.claimData(2);
-        gameProxy.attack{ value: bond }(disputed, 2, claim);
+        gameProxy.attackV2{ value: bond }(disputed, 2, claim, 0);
         (,,,,,, clock) = gameProxy.claimData(3);
         assertEq(clock.raw(), LibClock.wrap(Duration.wrap(25), Timestamp.wrap(uint64(block.timestamp))).raw());
 
         vm.warp(block.timestamp + 10);
-        bond = _getRequiredBond(3);
+        bond = _getRequiredBondV2(3, 0);
```

Comment:
As in OP's [bond spec](https://specs.optimism.io/fault-proof/stage-one/bond-incentives.html):
> FPM bonds are priced in the amount of gas that they are intended to cover. Bonds start at the very first depth of the game at a baseline of 400_000 gas. The 400_000 value is chosen as a deterrence amount that is approximately double the cost to respond at the top level.
Due to more claims (~2048 claims) should be submitted in multi-sec fault proof, maybe we should double check the gas at the top level remains ~200_000 gas cost later?